Mathlib/CategoryTheory/Limits/Shapes/Diagonal.lean

```diff
@@ -99,7 +99,34 @@ theorem pullback_diagonal_map_snd_snd_fst :
 
 variable [HasPullback i₁ i₂]
 
-set_option maxHeartbeats 400000 in
+/-- The underlying map of `pullbackDiagonalIso` -/
+abbrev pullbackDiagonalMapIso_hom :
```

Comment:
The name should not contain `_`. I would rather make these definitions rather than abbrev.
```suggestion
def pullbackDiagonalMapIso.hom :
```

Reply:
The point of these was just to make `pullbackDiagonalMapIso` quicker (as it needed an increase in maxHeartBeats). If I make it a def I will need to add simp lemmas as well, and wouldn't that make `pullbackDiagonalMapIso` more annoying to work with (because you need to unfold the map twice?)